### PR TITLE
Update to sccache v0.7.7

### PR DESCRIPTION
**Pull request by trxcllnt**
Update sccache to fix https://github.com/mozilla/sccache/issues/2038

### CODE REVIEW

**Review by bdice — COMMENTED** [review]
Looking at the last update: https://github.com/rapidsai/devcontainers/commit/aba251dc6e9c4f7679752f5bf3bee916792d825c.

Do we also need to update `features/src/sccache/devcontainer-feature.json` to remove `0.7.4` as a candidate feature version, and replace it with `0.7.7`?

**Comment by KyleFromNVIDIA** [comment]
IMO, there should be no trace of `0.7.4` or `0.7.5` being allowed to be used *anywhere*. We should replace it.

File: features/src/sccache/devcontainer-feature.json
Comment (bdice) [inline features/src/sccache/devcontainer-feature.json:1]: Does this feature need to have its version bumped?

**Review by bdice — APPROVED** [review]

File: features/src/sccache/devcontainer-feature.json
Comment (trxcllnt) [reply features/src/sccache/devcontainer-feature.json:1]: Yes it does.

**Pull request merged**